test/testdata/interface_axelar_gateway_gateway.json

```diff
@@ -15,19 +15,6 @@
           "type": "Versioned"
         }
       ]
-    },
-    "CreatorCap": {
```

Comment:
this might affect the sui deployment scripts, please update that as well

Reply:
https://github.com/axelarnetwork/axelar-contract-deployments/pull/447 (will only work after we merge this)